test/util_test.ml

```diff
@@ -0,0 +1,211 @@
+open Alcotest
+open Geneweb
+
+let mutil_contains () =
+  let str = "foo bar" in
+  let test t b = (check bool) t b (Mutil.contains str t) in
+  test "foo" true;
+  test "baz" false;
+  test "foo_b" false;
+  test "foo b" true;
+  test "foo__b" false;
+  test "bar__" false;
+  test "r" true;
+  test "" true
+
+let mutil_start_with () =
+  check_raises "" (Invalid_argument "start_with") (fun () ->
+      ignore @@ Mutil.start_with "foo" (-1) "foo");
+  check_raises "" (Invalid_argument "start_with") (fun () ->
+      ignore @@ Mutil.start_with "foo" 4 "foo");
+  (check bool) "Mutil.start_with \"foo\" 0 \"foo\"" true
+    (Mutil.start_with "foo" 0 "foo");
+  (check bool) "not (Mutil.start_with \"bar\" 0 \"foo\")" true
```

Comment:
I guess at some point we would like to use `(check bool) {|not (Mutil.start_with "bar" 0 "foo")|} true` to make the code more readable ?